flow.py

```diff
@@ -142,6 +144,8 @@ def create_packet(self, packet_id, delay=0.0):
         event1_message = "flow.create_packet: Flow" + str(self.flow_id) + \
             ": made data packet " + str(new_packet.packet_id)
         self.ns.add_event(event1, event1_message, delay=delay)
+        self.ns.record_packet_send_time(self.flow_id, new_packet.packet_id)
+        self.ns.record_flow_rate(self.flow_id, new_packet.packet_size)
```

Comment:
Does the flow rate only consider when packets leave from the source or is it supposed to contain info on all packets in the flow wherever they are?